cpp/ql/src/Likely Bugs/Memory Management/SuspiciousCallToStrncat.ql

```diff
@@ -1,26 +1,69 @@
 /**
  * @name Potentially unsafe call to strncat
- * @description Calling 'strncat' with the size of the destination buffer
- *              as the third argument may result in a buffer overflow.
+ * @description Calling `strncat` with the size of the destination buffer as the third argument may result in a buffer overflow.
```

Comment:
The description is a bit long but happy for you to leave it as is if there's no workaround

Reply:
Thanks! Fixed in 5382ef7761c32025a5a63344934ae51330ddbf6d.